src/plugins/intel_gpu/src/runtime/layout.cpp

```diff
@@ -239,6 +239,8 @@ static format to_weights_format(format f, bool is_grouped) {
             return format::o_is_yx_isv16;
         case format::bs_xs_xsv8_bsv8:
             return format::os_i_osv8__ai8;
+        case format::b_fs_yx_32fp:
+            return format::os_is_yx_osv32_isv32p;
```

Comment:
Is this `b_fs_yx_32fp -> os_is_yx_osv32_isv32p` mapping correct? As I understand blocking by dimensions does not match between them.